composer.json

```diff
@@ -13,7 +13,7 @@
         "ext-json": "*",
         "ext-soap": "*",
         "cweagans/composer-patches": "^1.6",
-        "kontrolgruppen/core-bundle": "1.11.5",
+        "kontrolgruppen/core-bundle": "dev-feature-KON-428-process-clients",
```

Comment:
I can see where this is coming from, but this is probably not what we want in the develop branch.